recipes/tximport-scripts/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: tximport-scripts
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/tximport-scripts/archive/v{{ version }}.tar.gz 
+  sha256: 7910a7519dd6655d8a2007019639d4204829aeb8d1a9f6a94d0c80055003ea10 
+
+build:
+  number: 0
+  skip: true  # [win32]
+  noarch: generic
+
+requirements:
+    host:
+        - r-base 3.5.1
```

Comment:
You should be able to just use `r-base`, since (A) we only build for R 3.5.1 at the moment and anyway the bioconductor packages that are run time dependencies are themselves pinned to specific R versions.